### DISCUSSION TITLE
E2s Factory Samples on Blue Hacktribe

### DISCUSSION THREAD

**Discussion by xCBNZ** [bangcorrupt/hacktribe#306] [Q&A] [answered]
Hey, i wanted to ask if it is possible to put the E2s Factory Samples on my Blue Hacktribe and if it is where do i get the samples from?

**Comment by bangcorrupt** [comment, marked as answer]
Yes, you need to export them from a sampler then edit the file.  See [Add factory samples from e2 sampler](https://github.com/bangcorrupt/hacktribe/wiki/How-To#add-factory-samples-from-e2-sampler) in the wiki and [this comment](https://github.com/bangcorrupt/hacktribe/discussions/73#discussioncomment-1961200).

**Reply by xCBNZ** [reply]
Hey, thanks but i already knew that. The problem is that i don't have a Sampler to get the samples

**Reply by zestafonsky** [reply]
Hey there, does it works for blue electribe 2 (synth version)? I stuck with exporting samples, as there nothing like that to export.

**Reply by bangcorrupt** [reply]
To get the samples from the Synth, follow [this process](https://github.com/bangcorrupt/hacktribe/wiki/Dump-Synth-PCM-Samples).  You can use the [Hacktribe Tutorial]() to get the firmware update file.    Also read #77 and https://github.com/bangcorrupt/hacktribe-tutorial/issues/2.

**Reply by xCBNZ** [reply]
There is a method to get the Blue Factory Samples but it seems very hard to me (in terms of coding) I got the e2s sample data from the Sampler. Now im using Blue hacktribe with Factory samples from Red Electribe

**Reply by zestafonsky** [reply]
Hacktribe Tutorial's e2s sample data from the Sampler worked, but it now default patterns doesn't sound well. Is there any fix for that?

**Reply by bangcorrupt** [reply]
If you have the patterns from the sampler and the samples from the sampler, you can use [this script](https://github.com/bangcorrupt/hacktribe/blob/main/scripts/e2pat_shift.py) to edit the patterns.

See https://github.com/bangcorrupt/hacktribe/discussions/61#discussioncomment-4502488.

**Reply by zestafonsky** [reply]
thank you!

**Comment by BATMAN-YEAH** [comment]
hello , I would like to do the opposite, put the E2 sounds/samples on my E2S, possible?
